include/powsybl/iidm/Identifiable.hxx

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#ifndef POWSYBL_IIDM_IDENTIFIABLE_HXX
+#define POWSYBL_IIDM_IDENTIFIABLE_HXX
+
+#include <powsybl/iidm/Identifiable.hpp>
+
+namespace powsybl {
+
+namespace iidm {
+
+template <typename T>
+bool Identifiable::isInstanceOf(const Identifiable& identifiable) {
+    return stdcxx::isInstanceOf<T>(identifiable);
+}
```

Comment:
Why do you need to duplicated this feature?

Reply:
I could find out how to specialize getAll<Connectable, Connectable> by using directly the stdcxx version : 
```
template <>
stdcxx::const_range<Connectable> NetworkIndex::getAll<Connectable, Connectable>() const {
    return boost::adaptors::values(m_objectsById) | boost::adaptors::indirected | boost::adaptors::filtered(stdcxx::isInstanceOf<Connectable, Identifiable>);
}
```

The compiler error :
```/mnt/e/Projet_SRO/rte/powsybl-iidm4cpp/src/iidm/NetworkIndex.cpp:60:83: error: no matching function for call to object of type 'const range_detail::forwarder<range_detail::filter_holder>' [clang-diagnostic-error]
    return boost::adaptors::values(m_objectsById) | boost::adaptors::indirected | boost::adaptors::filtered(stdcxx::isInstanceOf<Connectable, Identifiable>);
                                                                                  ^
/usr/include/boost/range/adaptor/argument_fwd.hpp:45:23: note: candidate template ignored: couldn't infer template argument 'T'
            Holder<T> operator()( T t ) const
                      ^
```